largestPrimePalindrome/largestPrimePalindrome.js

```diff
@@ -6,3 +6,25 @@
  * Extra credit: Modify the function to look for the largest prime palindrome less than any provided number.
  * Extra credit 2: How can you improve the time complexity?  (Google: Sieve of Eratosthenes)
  */
+
+const largestPrimePalindrome = () => {
+  
+  const isPalindrome = num => {
```

Comment:
I love the use of building out functions that separate the concerns! This is a wonderful solution! keep it up!